openapi/nexus.json

```diff
@@ -20131,10 +20131,20 @@
         "type": "object",
         "properties": {
           "metric_type": {
-            "$ref": "#/components/schemas/MetricType"
+            "description": "The type of this metric.",
```

Comment:
Interesting, I wonder why this changed?

Reply:
See https://github.com/oxidecomputer/omicron/pull/6364#discussion_r1720579988.